source/shared/cpp/ObjectModel/MarkDownParser.cpp

```diff
@@ -43,6 +43,11 @@ bool MarkDownParser::IsEscaped() const
     return m_isEscaped;
 }
 
+std::string MarkDownParser::GetText() const
+{
+    return m_text;
```

Comment:
is it the case that this is un-modified text? if so, consider renaming to `MarkDownParser::GetRawText()`

Reply:
Understood & thank you.